docs/src/pages/[platform]/components/autocomplete/index.page.mdx

```diff
@@ -28,7 +30,7 @@ export async function getStaticProps() {
 <PageTabLayout 
   tabComponents={[{
     title: "Documentation", 
-    children: <Fragment>{({ platform }) => import(`./${platform}.mdx`)}</Fragment>
+    children: <ReactPage />
```

Comment:
From a cursory glance, it looks like every primitive file repeats:
```

<PageTabLayout 
  tabComponents={[{
    title: "Documentation", 
    children: <ReactPage />
  },
  {
    title: "Props", 
    children: <PropsTableTab 
                componentName={frontmatter.title} 
                PropsData={data} 
                htmlElement={frontmatter.htmlElement} 
                mdnUrl={frontmatter.mdnUrl}
              />
  }]}
/>
```

Is that correct and if so can we just have a single component reused in each of these files?

Reply:
@jacoblogan Can you please address this question?

Reply:
Yes every primitive file is repeating this pattern but the abstraction is already in place with `PageTabLayout` while we could remove a little more by creating another abstraction it feels like this wouldn't be worth it because `<ReactPage />`, `data`, and all of the `frontmatter` information is coming from the page itself so these would still need to be passed into the newly created component.

Reply:
Messaged you offline